Makefile

```
@@ -0,0 +1,43 @@
+# For more on Makefiles: https://www.gnu.org/software/make/manual/make.html
+
+# Phony targets (targets that don't represent files)
+.PHONY: all clean help
+
+# Default target - runs when you type 'make' with no arguments
+all:
+	@echo "Available targets: all, clean, help"
+	@echo "Run 'make help' for more information"
+
+# Help - displays available targets and their descriptions
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (8).

(maxbodylength)